pkg/clusterapi/etcd.go

```diff
@@ -0,0 +1,86 @@
+package clusterapi
+
+import (
+	etcdbootstrapv1 "github.com/aws/etcdadm-bootstrap-provider/api/v1beta1"
+	etcdv1 "github.com/aws/etcdadm-controller/api/v1beta1"
+	v1 "k8s.io/api/core/v1"
+	clusterv1 "sigs.k8s.io/cluster-api/api/v1beta1"
+	bootstrapv1 "sigs.k8s.io/cluster-api/bootstrap/kubeadm/api/v1beta1"
+	controlplanev1 "sigs.k8s.io/cluster-api/controlplane/kubeadm/api/v1beta1"
+
+	"github.com/aws/eks-anywhere/pkg/api/v1alpha1"
+	"github.com/aws/eks-anywhere/pkg/cluster"
+	"github.com/aws/eks-anywhere/pkg/constants"
+)
+
+// SetEtcdConfigInEtcdCluster sets up the etcd config in EtcdadmCluster.
+func SetEtcdConfigInEtcdCluster(etcd *etcdv1.EtcdadmCluster, version string) {
```

Comment:
Same comment about all these "Set" methods, I would vote to make them unexported and test them through the main method that uses them

Reply:
same as above